.pre-commit-config.yaml

```diff
@@ -6,6 +6,7 @@ ci:
   autofix_commit_msg: 'style: pre-commit.ci fixes'
   skip:
     - poetry-lock
+    - search-and-replace
```

Comment:
Let's see what https://github.com/mattlqx/pre-commit-search-and-replace/issues/35 brings